src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -783,6 +793,22 @@ private static unsafe void RemoveSessionCallback(IntPtr ctx, IntPtr session)
             ctxHandle.RemoveSession(name);
         }
 
+#if DEBUG
+        [UnmanagedCallersOnly]
+        private static unsafe void KeyLogCallback(IntPtr ssl, char* line)
+        {
+            ReadOnlySpan<byte> data = MemoryMarshal.CreateReadOnlySpanFromNullTerminated((byte*)line);
+            if (s_fileStream != null && data.Length > 0)
```

Comment:
This s_fileStream != null should always be true, or else we wouldn't have registered the callback, right? Can we just assert it instead?

Reply:
yes, I can add Assert. I added the if becasue of nullability check. I can override it  with `!` + assert but I was not sure if it matters for debug build.

Reply:
`Debug.Assert(s_fileStream != null)` will also suppress nullable reference types warnings from the compiler, if that's what you're referring to.